app/mailers/donation_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class DonationMailer < ApplicationMailer
+
+  def new_donation_email(donation)
+    @donation = donation
+    @organization = @donation.organization
+    recipients = Rails.application.secrets.donation_email_recipients.split(',')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.